profiles/prefix/linux/riscv/make.defaults

```diff
@@ -0,0 +1,11 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+ARCH="riscv"
+CHOST="riscv64-pc-linux-gnu"
+# The base profile sets ACCEPT_KEYWORDS=riscv and we don't have that in prefix.
+ACCEPT_KEYWORDS="~riscv"
+
+# We don't have lib64 in prefix so, remove it here.
+SYMLINK_LIB=""
+LIBDIR_riscv="lib"
```

Comment:
I think this needs to be either `LIBDIR_lp64="lib"` or `LIBDIR_lp64d="lib"`, see `default/linux/riscv/20.0/rv64gc/lp64d/musl/make.defaults` for example.